src/OrchardCore/OrchardCore/Localization/BclCalendars.cs

```diff
@@ -20,8 +20,8 @@ public static CalendarSystem GetCalendarByName(CalendarName calendarName) =>
             CalendarName.Gregorian => CalendarSystem.Iso,
             CalendarName.Julian => CalendarSystem.Julian,
             CalendarName.Persian => CultureInfo.CurrentUICulture.Calendar.IsLeapYear(1)
-                ? CalendarSystem.PersianSimple
-                : CalendarSystem.PersianAstronomical,
+            ? CalendarSystem.PersianSimple
```

Comment:
I think there's a format issue here

Reply:
I am not going to argue with formatting. I follow the placement of the cursor.

Reply:
There is no issue with formatting. Let's not waste anytime with formatting-related comments please.